sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -168,7 +168,8 @@ def test_invalid_json_neg_inf(self):
 @unittest.skipIf(HttpError is None, 'GCP dependencies are not installed')
 class TestBigQuerySource(unittest.TestCase):
   def test_display_data_item_on_validate_true(self):
-    source = beam.io.BigQuerySource('dataset.table', validate=True)
+    source = beam.io.BigQuerySource(
```

Comment:
How many of these tests fail if we don't force the native source?

Reply:
Tests in this file use the native BQ source implementation (e.g. specific display data, using the Reader implementation used for DirectRunner, etc. I think after deprecation, most of the tests in this particular class would go away) - integration tests are not having to change almost at all.

Reply:
I assume we have the same test coverage for the new source ?

Reply:
yes, most tests have variants for the new and the old source running alongside each other.